packages/react-core/src/components/WizardComposable/WizardComposableBody.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import styles from '@patternfly/react-styles/css/components/Wizard/wizard';
+import { css } from '@patternfly/react-styles';
+
+/**
+ * Used as a "main" wrapper for WizardComposableStep content, where the wrapping element is customizable.
+ */
+
+export interface WizardComposableBodyProps {
+  children?: React.ReactNode | React.ReactNode[];
+  /** Set to true to remove the default body padding */
+  hasNoBodyPadding?: boolean;
+  /** An aria-label to use for the wrapper element */
+  'aria-label'?: string;
+  /** Sets the aria-labelledby attribute for the wrapper element */
+  'aria-labelledby'?: string;
+  /** Component used as the wrapping content container */
+  wrapperElement?: React.ElementType;
+}
+
+export const WizardComposableBody = ({
+  children,
+  hasNoBodyPadding = false,
+  'aria-label': ariaLabel,
+  'aria-labelledby': ariaLabelledBy,
+  wrapperElement: Wrapper = 'div'
+}: WizardComposableBodyProps) => (
+  <Wrapper
+    aria-label={ariaLabel}
+    aria-labelledby={ariaLabelledBy}
+    className={css(styles.wizardMain)}
+    {...(Wrapper === 'div' && { role: 'main' })}
```

Comment:
I don't believe we should set `role="main"` on the component anywhere, just because it's best practice to only include one `main` role/element.

Reply:
I think this is what inspired the idea: https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/components/Wizard/WizardToggle.tsx#L96, but I can see where some divs might not want that role defined.

Reply:
Ah interesting. So in terms of a modal-like Wizard, setting a `main` within the Wizard can work as long as we hide any other `main` elements/roles on the page. Testing the [basic wizard example](https://www.patternfly.org/v4/components/wizard/#basic) out a little, I added the `isOpen` prop to it to get a `<main>` element rendered in the wizard body, then ran aXe and didn't get any errors regarding any `main` element. 

Once I removed `aria-hidden="true"` on the `#root` div, though, aXe flagged some things regarding the two `main` elements:

![Modal wizard elements pane](https://user-images.githubusercontent.com/70952936/181350788-d2d34221-df4f-4170-9bb0-1089949e73b4.png)

Reply:
Thanks for looking into that, because I was curious as well.